src/store/actions/Chat.ts

```diff
@@ -154,8 +159,34 @@ export const sendTableChat = createAsyncThunk<void, SendTableChatFields>(
   }
 );
 
+type ChatSidebarVisibilityAction = {
+  type: ChatActionTypes.SET_CHAT_SIDEBAR_VISIBILITY;
+  payload: boolean;
+};
```

Comment:
- https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters
  - > we recommend trying to treat actions more as "describing events that occurred", rather than "setters". Treating actions as "events" generally leads to more meaningful action names, fewer total actions being dispatched, and a more meaningful action log history. Writing "setters" often results in too many individual action types, too many dispatches, and an action log that is less meaningful.
  
This can be called out in many places, but I will just comment in this one place at the moment. Consider applying it across the entire PR though, as it will reduce the future re-work we have to do to clean things up later.